src/scanner.c

```diff
@@ -306,3 +337,13 @@ ts_scanner_scan_one(ScannerCtx *ctx, bool fail_if_not_found, char *item_type)
 			return false;
 	}
 }
+
+TSDLLEXPORT ItemPointer
```

Comment:
We usually don't include exporting details such as this macro into C files, but this file seems to be a complete exception :)

Reply:
Yes, this was also the reason I added it here. I figured a separate PR can clean this up if necessary